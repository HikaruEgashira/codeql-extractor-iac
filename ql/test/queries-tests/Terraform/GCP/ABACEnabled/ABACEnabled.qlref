security/Terraform/GCP/ContainerCluster/ABACEnabled.ql