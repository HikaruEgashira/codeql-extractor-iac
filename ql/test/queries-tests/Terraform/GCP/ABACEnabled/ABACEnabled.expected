| rbac_enabled.hcl:6:24:6:27 | true | ABAC Enabled |
